extensions/firefox/content/pdfjschildbootstrap-enabled.js

```diff
@@ -0,0 +1,31 @@
+/* Copyright 2014 Mozilla Foundation
```

Comment:
That's good to know. I have some other changes I want to make to avoid loading .jsms when no PDF has been loaded, and it looks like that will help me.